CURSOR.md

```
@@ -0,0 +1,235 @@
+
+### Quick start
+- Show all make targets:
+```bash
+make help
+```
+- Development (Docker, recommended):
+```bash
+make start
+# App: http://localhost:3000
+```
+- Development (host machine, no Docker):
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)